commands/create_command.py

```diff
@@ -0,0 +1,29 @@
+import json
+import os
+from keys import keys
+from commands.abstract_command import ICommand
+
+class CreateCommand(ICommand):
+
+    def __init__(self):
+        with open('schema.json','r') as schema:
```

Comment:
why you hardcoded the schema path? it should be an argument taken as an input.
Besides, you should validate the schema file.

Reply:
+1

Reply:
What do you mean by hardcoding the path?
And yes we will validate the schema

Reply:
Hard coded path mean you are assuming the file name is schema.json and reside with the same folder of py file.

you take the path as a variable